resources/views/tasks/parts/row.blade.php

```diff
@@ -0,0 +1,35 @@
+<tr>
+    <td><span data-bs-toggle="tooltip" data-bs-placement="top" title="{{ $task->created_at->toEuropeanDateTime() }}">{{ $task->created_at->diffForHumans() }}</span></td>
+    <td><a href="{{ route('user.show', $task->reference_user_id) }}">{{ \App\Models\User::find($task->reference_user_id)->name }} ({{ $task->reference_user_id }})</a></td>
+    <td>
+        <i class="fas {{ $task->type()->getIcon() }}" data-bs-toggle="tooltip" data-bs-placement="top" title="{{ $task->type()->getName() }}"></i>
+
+        @if($task->type()->getLink($task))
+            <a href="{{ $task->type()->getLink($task) }}" target="_blank" class="link-offset-1 dotted-underline">{{ $task->type()->getText($task) }}</a>
+        @else
+            {{ $task->type()->getText($task) }}
+        @endif
+    </td>
+    <td>
+        @isset($task->sender_user_id)
+            <a href="{{ route('user.show', $task->sender_user_id) }}">{{ \App\Models\User::find($task->sender_user_id)->name }} ({{ $task->sender_user_id }})</a>
+        @else
+            System
+        @endisset
+    </td>
+    
+    <td>
+        @if($activeFilter != 'sent' && $activeFilter != 'archived')
+            <a href="{{ route('task.complete', $task->id) }}" class="btn btn-sm btn-outline-success text-decoration-none"><i class="fas fa-check"></i> Complete</a>
+            <a href="{{ route('task.decline', $task->id) }}" class="btn btn-sm btn-outline-danger text-decoration-none"><i class="fas fa-xmark"></i> Decline</a>
```

Comment:
suggestion: separate complete and decline, and make decline smaller, but available through aria. remove pointers to `->id`, laravel looks these up itself

```suggestion
        @if(!in_array($activeFilter, ['sent', 'archived']))
            <div class="btn-toolbar justify-content-between" role="toolbar" aria-label="Task actions">
            <div class="btn-group">
                <a href="{{ route('task.complete', $task) }}" class="btn btn-sm btn-outline-success" title="Complete task" role="button"><i class="fas fa-check"></i> Complete</a>
                </div>
            <div class="btn-group">
                <a href="{{ route('task.decline', $task->id) }}{{ route('task.decline', $task) }}" class="btn btn-sm btn-outline-danger text-decoration-none" title="Decline task" role="button">
                    <i class="fas fa-xmark"></i><span class="visually-hidden">Decline</span>
                </a>
            </div>
        </div>
```

Reply:
Implemented a tweaked version of this. Mostly because it started looking weird without the column title and the decline being so far away. So decline is next to complete, but added "are you sure you want to.." dialog.